src/components/AccountFormFields/index.tsx

```diff
@@ -65,6 +61,14 @@ function AccountFormFields({ showPasswordOnLoad }: AccountFormFieldsProps) {
     setShowPassword(false);
   }, []);
 
+  const validatePassword = (val) => {
```

Comment:
were there unit tests for validation that needed to be updated?

Reply:
I'd also add a comment here explaining the forbidding of the period, just for future developers since it seems like a strange thing.  I know you explained in the body of the PR but I'm sure that's something that will eventually become lost to history.